asm/benchRptAxpy.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+go get -u github.com/Kunde21/sift
+
+go test ./...  -bench Axpy | tee >( sift markL | sed s/markL/mark/ > old.tst ) |  sift mark[^L] >new.tst
+benchcmp old.tst new.tst
```

Comment:
One full benchmark takes 20+ minutes, so I've been using benchcmp to keep the time down.
[Here's a 10x benchstat run](https://github.com/gonum/internal/files/318985/bnchRpt.txt) that took a few hours to build.